### PR TITLE
refactor to use Timing context

### PR DESCRIPTION
**Pull request by aaron-prindle**

### CODE REVIEW

**Comment by aaron-prindle** [comment]
waiting on phase1.5-python PR to change python/builder.py

File: ftl/common/ftl_util.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  def __init__(self, descriptor):
+    self.descriptor = descriptor
+
+  def __enter__(self):
+    self.start = time.time()
+    return self
+
+  def __exit__(self, unused_type, unused_value, unused_traceback):
+    end=time.time()
+    print '%s took %d seconds' % (self.descriptor, end-self.start)
```
Comment (dlorenc) [inline ftl/common/ftl_util.py]: Could this use a logger or something more configurable than print?

File: ftl/php/builder.py [hunk 2]
```diff
@@ -71,24 +72,22 @@ def _gen_package_tar(self, destination_path):
...
             ['rm', '-rf', os.path.join(app_dir, 'vendor')])
-        logging.info('Starting composer install ...')
-        subprocess.check_call(
-            ['composer', 'install', '--no-dev', '--no-scripts'], cwd=app_dir)
-        logging.info('Finished composer install.')
 
-        logging.info('Starting to tar composer packages...')
-        subprocess.check_call(['tar', '-C', tmp, '-cf', tar_path, '.'])
-        logging.info('Finished generating tarfile for composer packages.')
+        with ftl_util.Timing("composer install"):
```
Comment (dlorenc) [inline ftl/php/builder.py]: For searching later it might be nice to make these descriptors all have _ instead of spaces.

Comment (aaron-prindle) [reply ftl/php/builder.py]: done

File: ftl/common/ftl_util.py (hunk 1 above)
Comment (aaron-prindle) [reply ftl/common/ftl_util.py]: done

**Comment by dlorenc** [comment]
Looks like a line length/flake issue.

**Pull request merged**